enforcer/src/main/resources/Dockerfile

```diff
@@ -67,5 +68,8 @@ COPY maven/conf/log4j2.properties conf/log4j2.properties
 COPY maven/security security
 COPY check_health.sh .
 COPY maven/LICENSE.txt .
+COPY ./applicationinsights.json  conf/
+COPY ./applicationinsights-agent-3.1.1.jar conf/
 
-CMD java -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath="${ENFORCER_HOME}/logs/heap-dump.hprof" $JAVA_OPTS -Dlog4j.configurationFile="${ENFORCER_HOME}/conf/log4j2.properties" -cp "lib/*:lib/dropins/*" org.wso2.choreo.connect.enforcer.server.AuthServer
+
+CMD java -javaagent:/home/wso2/conf/applicationinsights-agent-3.1.1.jar -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath="${ENFORCER_HOME}/logs/heap-dump.hprof" $JAVA_OPTS -Dlog4j.configurationFile="${ENFORCER_HOME}/conf/log4j2.properties" -DtracingEnabled="true" -cp "lib/*:lib/dropins/*" org.wso2.choreo.connect.enforcer.server.AuthServer
```

Comment:
Shall we place this in lib directory (rather than using conf dir)? @Rajith90 wdyt?

Reply:
ah since we are asking to mount the entire lib directory (due to the way lib directory provided in docker compose file), I think we may need some other directory here.

Reply:
btw `applicationinsights-agent-3.1.1.jar` jar is comitted to the PR

Reply:
We need to decide on placing the agent jar and mounting the applicationinsights.json file. I have removed this JVM arg and added a todo section in the dockerfile to track this. For tracking purposes I have commited the agent jar and its config file with this PR. It should not be an issue as we do not pack either the jar or the config file.